test/views/businessactivities/expected_amls_turnoverSpec.scala

```diff
@@ -48,7 +47,7 @@ class expected_amls_turnoverSpec extends AmlsViewSpec with MustMatchers  {
 
       val form2: ValidForm[ExpectedAMLSTurnover] = Form2(ExpectedAMLSTurnover.Third)
 
-      def view = views.html.businessactivities.expected_amls_turnover(form2, true, businessMatching)
+      def view = views.html.businessactivities.expected_amls_turnover(form2, true, None, Some(List("accountancy service provider")))
```

Comment:
Any reason you are replacing businessMatching with None?

Reply:
^